packages/toolkit/src/dev/activation.ts

```diff
@@ -152,6 +153,18 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
     if (!isCloud9() && !isReleaseVersion() && config.betaUrl) {
         ctx.subscriptions.push(watchBetaVSIX(config.betaUrl))
     }
+
+    if (DevSettings.instance.isNewLoginEnabled()) {
+        void vscode.commands.executeCommand('setContext', 'aws.explorer.newLoginEnabled', true)
```

Comment:
Would be cool if DevSettings automatically set `aws.dev.xx` by default (for all "dev settings"), then we wouldn't need this.

Reply:
I did not find an easy way to do this. My next priority is to understand how auth can be shared across extensions. I can get back to this later.